### PR TITLE
dBay integration

### PR DESCRIPTION
**Pull request by LuisEGV**
Added Feature: I added the integration for dBay, a dApp marketplace (https://www.dbay.ai/).

The changes were based on the dApp integration documentation: https://status.im/developer_tools/add_your_dapp.html

### CODE REVIEW

**Comment by oskarth** [comment]
Not sure why this PR was left hanging for 10 days, pinged people in dapp team. Sorry about the delayed response @LuisEGV!

File: src/status_im/ui/screens/browser/default_dapps.cljs [hunk 1]
```diff
@@ -16,6 +16,10 @@
              :dapp-url    "https://app.ercdex.com/"
              :photo-path  "contacts://erc-dex"
              :description "Trustless trading has arrived on Ethereum"}
+             {:name        "dBay"
```
Comment (rachelhamlin) [inline src/status_im/ui/screens/browser/default_dapps.cljs]: Perhaps dBay would be a better fit under `Marketplaces`?

File: src/status_im/ui/screens/browser/default_dapps.cljs [hunk 2]
```diff
@@ -16,6 +16,10 @@
...
+             :dapp-url    "https://www.dbay.ai/"
+             :photo-path  "contacts://dBay"
+             :description "Buy all Yours dApps in one place."}
```
Comment (rachelhamlin) [inline src/status_im/ui/screens/browser/default_dapps.cljs]: Suggested copy tweak: `Buy from all your favorite DApps in one place.` You guys trade ERC721s, right, @LuisEGV?

**Review by rachelhamlin — CHANGES_REQUESTED** [review]

**Comment by rachelhamlin** [comment]
Thanks @LuisEGV! I left a few comments.

**Review by flexsurfer — APPROVED** [review]

**Review by rachelhamlin — APPROVED** [review]

File: src/status_im/ui/screens/browser/default_dapps.cljs (hunk 2 above)
Comment (LuisEGV) [reply src/status_im/ui/screens/browser/default_dapps.cljs]: I just changed this commit to fit the description you suggested and also included dBay in the marketplace section. Thanks for that Rachel :)

Comment (rachelhamlin) [reply src/status_im/ui/screens/browser/default_dapps.cljs]: Thanks for the changes @LuisEGV!

**Comment by LuisEGV** [comment]
Hey, guys (@rachelhamlin @flexsurfer ) are their things that I need to change for this commit to go through. @jeluard is there something you want me to change?

**Comment by flexsurfer** [comment]
@LuisEGV could you please rebase onto develop and squash commits? thanks

**Comment by StatusSceptre** [comment]
@LuisEGV Any update on rebase and squash commits?

**Comment by LuisEGV** [comment]
> @LuisEGV Any update on rebase and squash commits?
Hi StatusSpectre! I am having some difficulties with rebasing to my first commit because I committed with different accounts. Is there any possibility you guys could go over the last three commits and see if it will work rather than squashing.

**Comment by jeluard** [comment]
@LuisEGV You could recreate a new PR?

**Comment by LuisEGV** [comment]
> @LuisEGV You could recreate a new PR?

Yeah, I think that's the best thing to do 👍

**Comment by LuisEGV** [comment]
The new pull request is here: https://github.com/status-im/status-react/pull/7706 👍

**Pull request closed without merging**